diff -Naur stable.orig/src/common/timekprpam.py stable/src/common/timekprpam.py
--- stable.orig/src/common/timekprpam.py	2018-10-07 01:49:40.435851000 +0300
+++ stable/src/common/timekprpam.py	2018-10-07 02:57:13.261278418 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """ A library timekpr uses to read/edit Linux-PAM configuration files.
     Currently using modules: time, access
     Warning: Not all Linux-PAM possibilities are supported!
diff -Naur stable.orig/src/timekpr-client.py stable/src/timekpr-client.py
--- stable.orig/src/timekpr-client.py	2018-10-07 01:49:40.435851000 +0300
+++ stable/src/timekpr-client.py	2018-10-07 12:25:39.067588788 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 # generic import
 import gi
@@ -74,8 +74,8 @@
     path.append('.')
     print("%s: development config" % ("main"))
 
-from common.timekprpam import *
-from common.timekprcommon import *
+from timekprpam import *
+from timekprcommon import *
 
 # timekpr.conf variables (dictionary variable)
 VAR = getvariables(DEVACTIVE)
diff -Naur stable.orig/src/timekpr-gui3.py stable/src/timekpr-gui3.py
--- stable.orig/src/timekpr-gui3.py	2018-10-07 01:49:40.435851000 +0300
+++ stable/src/timekpr-gui3.py	2018-10-07 03:05:23.427180980 +0300
@@ -1,15 +1,17 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """ The graphical user interface for timekpr configuration.
     Copyright / License: See COPYRIGHT.txt
 """
 
 import re
+import gi
 from os import remove, mkdir, geteuid, getenv
 from os.path import isdir, isfile, realpath, dirname, join
 from time import strftime, sleep
 from pwd import getpwnam
 from spwd import getspall
 
+gi.require_version("Gtk", "3.0")
 from gi.repository import Gtk
 from gi.repository import GObject
 
@@ -25,8 +27,8 @@
     from sys import path
     path.append('.')
 
-from common.timekprpam import *
-from common.timekprcommon import *
+from timekprpam import *
+from timekprcommon import *
 
 #timekpr.conf variables (dictionary variable)
 VAR = getvariables(DEVACTIVE)
diff -Naur stable.orig/src/timekpr-gui.py stable/src/timekpr-gui.py
--- stable.orig/src/timekpr-gui.py	2018-10-07 01:49:40.435851000 +0300
+++ stable/src/timekpr-gui.py	2018-10-07 03:00:44.284375432 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """ The graphical user interface for timekpr configuration.
     Copyright / License: See COPYRIGHT.txt
 """
@@ -38,8 +38,8 @@
 if DEVACTIVE:
     from sys import path
     path.append('.')
-from common.timekprpam import *
-from common.timekprcommon import *
+from timekprpam import *
+from timekprcommon import *
 
 #timekpr.conf variables (dictionary variable)
 VAR = getvariables(DEVACTIVE)
diff -Naur stable.orig/src/timekpr.py stable/src/timekpr.py
--- stable.orig/src/timekpr.py	2018-10-07 01:49:40.435851000 +0300
+++ stable/src/timekpr.py	2018-10-07 03:00:59.204311599 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 """ The "daemon service" for timekpr.
     Copyright / License: See COPYRIGHT.txt
 """
@@ -17,8 +17,8 @@
 if DEVACTIVE:
     from sys import path
     path.append('.')
-from common.timekprpam import * # timekprpam.py
-from common.timekprcommon import * # timekprcommon.py
+from timekprpam import * # timekprpam.py
+from timekprcommon import * # timekprcommon.py
 
 #timekpr.conf variables (dictionary variable)
 VAR = getvariables(DEVACTIVE)
