--- stable/support/bin/users.timekpr.orig	2016-09-28 21:52:47.076982000 +0300
+++ stable/support/bin/users.timekpr	2016-11-06 14:37:33.710440072 +0200
@@ -3,7 +3,7 @@
 # this filters out only Active users using loginctl
 ACTIVE_USERS_LOGINCTL()
 (
-	loginctl list-users | sed 's/^ \+//' | cut -d' ' -f2 | sort | uniq | \
+	loginctl list-users | sed "1,1d; $ d" | sed 's/^ \+//' | cut -d' ' -f2 | sort | uniq | \
 		while read TPKR_USER;
 		do
 			if  [ "$(loginctl show-user -p State $TPKR_USER)" = "State=active" ];
